backend/db.py

```diff
@@ -224,35 +224,33 @@ def get_rooms(user_id, org_id):
 def get_room_messages(room_id, org_id):
     helper = DataStorage()
     helper.organization_id = org_id
-    options = {"sort":{"created_at":-1}}
-    response = helper.read_query("dm_messages", query={"room_id": room_id}, options=options)
+    options = {"sort": {"created_at": -1}}
+    response = helper.read_query(
+        "dm_messages", query={"room_id": room_id}, options=options
+    )
     if response and "status_code" not in response:
         return response
     return []
 
 
 # get all the messages in a particular room filtered by date
-def get_messages(room_id,org_id, date):
+def get_messages(room_id, org_id, date):
```

Comment:
This filter can be made more generic by having `start_date` and `end_date`, rather than just `date`, both optional. If start date is not provided, then it defaults to `previous_day`. If `end_date` is not provided, then it defaults to `next_day`

Reply:
Thanks.
Though I may not be effect that change, because the scope of this PR is only restricted to sidebar codes refactoring.
It's a nice observation.